src/cljs/athens/electron/utils.cljs

```diff
@@ -61,12 +61,13 @@
 (defn self-hosted-db
   "Returns a map representing a self-hosted db.
    Self-hosted dbs are uniquely identified by the url."
-  [name url]
-  {:type   :self-hosted
-   :name   name
-   :id     url
-   :url    url
-   :ws-url (str "ws://" url "/ws")})
+  [name url password]
+  {:type     :self-hosted
+   :name     name
+   :id       url
+   :url      url
+   :password password
```

Comment:
We'll need to review the storage and transmission of sensitive information like passwords and tokens at some point. Out of scope for now, just wanted to mention it.

Reply:
Indeed